tutorials/metrics/plot_metrics.py

```diff
@@ -37,17 +47,17 @@
 raw.pick("eeg")
 raw.set_eeg_reference("average")
 
-#%%
-# Pycrostates implements several metrics to evaluate the quality of
-# clustering solutions without knowing the ground truth:
-# - The silhouette score (higher the better)
-# - The Calinski Harabasz score (higher the better)
-# - The Dunn score (higher the better)
-# - The Davies Bouldin score (lower the better)
-#
-# We can apply these metrics on clustering solution
-# computed with different values of n_clusters and
-# analyse which give the best clustering solution.
+# %%
+# In this exemple , we will use a single subject EEG recording in order to
+# study more give more explanation about each of this metrics
```

Comment:
'in order to study more give more explanation of'
Doesn't sound that great, could you rephrase a bit?